esmvaltool/recipes/mpqb/recipe_mpqb_xch4.yml

```diff
@@ -0,0 +1,307 @@
+# ESMValTool
+# recipe_mpqb_xch4.yml
+
+documentation:
+  title: Methane evaluation with ESA CMUG data
+  description: |
+    Recipe for the comparison of ESA XCH4 data with CMIP6 models
+
+  authors:
+    - hassler_birgit
+    - mueller_benjamin
+
+
+################################################
+# Define some default parameters using anchors #
+################################################
+
+nh_region: &nh_region
+  extract_region:
+    start_longitude: 0
+    end_longitude: 360
+    start_latitude: 30
+    end_latitude: 90
+
+sh_region: &sh_region
+  extract_region:
+    start_longitude: 0
+    end_longitude: 360
+    start_latitude: -90
+    end_latitude: -30
+
+mpqbregridder: &mpqbregridder
+  regrid:
+    target_grid: 5x5
+    scheme: area_weighted
+
+global_mon: &global_mon
+  custom_order: true
+  <<: *mpqbregridder
+  mask_multimodel:
+  area_statistics:
+    operator: mean
+  monthly_statistics:
+    operator: mean
+
+global_ann: &global_ann
+  custom_order: true
+  <<: *mpqbregridder
+  mask_multimodel:
+  area_statistics:
+    operator: mean
+  annual_statistics:
+    operator: mean
+
+sh_mon: &sh_mon
+  custom_order: true
+  <<: *mpqbregridder
+  <<: *sh_region
+  mask_multimodel:
+  area_statistics:
+    operator: mean
+  monthly_statistics:
+    operator: mean
+
+sh_ann: &sh_ann
+  custom_order: true
+  <<: *mpqbregridder
+  <<: *sh_region
+  mask_multimodel:
+  area_statistics:
+    operator: mean
+  annual_statistics:
+    operator: mean
+
+nh_mon: &nh_mon
+  custom_order: true
+  <<: *mpqbregridder
+  <<: *nh_region
+  mask_multimodel:
+  area_statistics:
+    operator: mean
+  monthly_statistics:
+    operator: mean
+
+nh_ann: &nh_ann
+  custom_order: true
+  <<: *mpqbregridder
+  <<: *nh_region
+  mask_multimodel:
+  area_statistics:
+    operator: mean
+  annual_statistics:
+    operator: mean
+
+
+# Datasets
+################################################
+
+datasets:
+### observations
+  - {dataset: CDS-XCH4, project: OBS, type: sat, version: L3, tier: 3,
+     derive: false, alias: CDS-XCH4}
+### CMIP6 (esm-hist)
+  # - {dataset: BCC-CSM2-MR, project: CMIP6, ensemble: r1i1p1f1, grid: gn,
+     # exp: esm-hist, derive: true, alias: BCC-CSM2-MR}
+  # - {dataset: CNRM-ESM2-1, project: CMIP6, ensemble: r1i1p1f2, grid: gr,
+     # exp: esm-hist, derive: true, alias: CNRM-ESM2-1}
+  # - {dataset: UKESM1-0-LL, project: CMIP6, ensemble: r1i1p1f2, grid: gn,
+     # exp: esm-hist, derive: true, alias: UKESM1-0-LL}
```

Comment:
Does the recipe work when you uncomment these datasets? If yes, please uncomment; if not, might be better to remove them.

Reply:
Yes, the recipe works also with these datasets. They are just commented out because you either look at the "historical" runs or the "esm-hist" runs, but normally not at both at the same time. However, since we decided to not look at these datasets in detail, I will delete these datasets in the recipe.